datapoint/datapoint.go

```diff
@@ -88,6 +88,39 @@ func (dp *Datapoint) String() string {
 	return fmt.Sprintf("DP[%s\t%s\t%s\t%d\t%s]", dp.Metric, dp.Dimensions, dp.Value, dp.MetricType, dp.Timestamp.String())
 }
 
+type metadata int
+
+const (
+	stringProperties metadata = iota
+)
+
+//SetProperty sets a property to be used when an MTS when it is created
```

Comment:
Replace "an MTS" with "the time series associated with the datapoint" to make it more system agnostic.